dll/cpl/desk/screensaver.c

```diff
@@ -594,6 +667,40 @@ OnInitDialog(HWND hwndDlg, PDATA pData)
         return FALSE;
     }
 
+    wc.lpfnWndProc = DefWindowProc;
+    wc.hInstance = hApplet;
+    wc.hCursor = NULL;
```

Comment:
`hCursor` is NULL?  Alright?

Reply:
Yes, seems to work fine on both Windows/ReactOS, also: https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rostests/tests/subclass/subclass.c;hb=826bd41d88df8b83494ce921f7f4a3ec55544c7f#l16

MSDN: https://docs.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-wndclassa

> If this member is NULL, an application must explicitly set the cursor shape whenever the mouse moves into the application's window.